runtime/src/iree/schemas/vulkan_executable_def.fbs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2019 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+include "iree/schemas/executable_debug_info.fbs";
+
+namespace iree.hal.vulkan;
+
+// 'Vulkan v1 Executable'.
+file_identifier "VKE1";
+file_extension "vk1";
+
+// Direct overlay of the VkPushConstantRange struct.
+struct PushConstantRange {
+  stage_flags:uint32;  // VkShaderStageFlags
+  offset:uint32;
+  size:uint32;
+}
+
+// Direct overlay of the VkDescriptorType enum.
+enum VkDescriptorType:uint32 {
+  SAMPLER = 0,                              // VK_DESCRIPTOR_TYPE_SAMPLER
```

Comment:
Do we (plan to) use sampler actually? Or is this for some other purposes?

Reply:
flatbuffers requires a 0 value so I had to have something and wanted to keep it the same as VkDescriptorType - though one day maybe :)